manager/container.go

```diff
@@ -306,8 +322,8 @@ func newContainerData(containerName string, memoryCache *memory.InMemoryCache, h
 	cont := &containerData{
 		handler:                  handler,
 		memoryCache:              memoryCache,
-		housekeepingInterval:     *HousekeepingInterval,
-		maxHousekeepingInterval:  maxHousekeepingInterval,
+		housekeepingInterval:     jitter(*HousekeepingInterval, JitterFactor),
```

Comment:
Doesn't that have the potential of skipping a housekeeping period as @vishh detailed above?